recipes/index.js

```diff
@@ -7,25 +7,26 @@ const {
 } = require("./utils");
 const install = require('./install');
 
-program.option("-f, --app-name <value>", "App Name");
-program.parse(process.argv);
+program.requiredOption("-c, --app-name <value>", "App Name");
+program.option("-d, --directory <value>", "Use directory");
+program.parse();
 
-//pre install check
-const appName = program.args[0];
```

Comment:
this is needed to run the command without `-c`. should work in all below cases.

```
npx react-chef <your-app-name> // without c
npx react-chef <your-app-name> -d <relative-path> // without c
npx react-chef -c <your-app-name>
npx react-chef -c <your-app-name> -d <relative-path>
```

Reply:
probably the option is not supported with commander need
to think about it. do you have any idea?

Reply:
resolved